tests/divvytests/conftest.py

```diff
@@ -3,23 +3,29 @@
 import looper.divvy as divvy
 import pytest
 
+from looper.divvy import select_divvy_config, DEFAULT_CONFIG_SCHEMA
+
 
 THIS_DIR = os.path.dirname(os.path.abspath(__file__))
 DATA_DIR = os.path.join(THIS_DIR, "data/divcfg-master")
 FILES = glob.glob(DATA_DIR + "/*.yaml")
-DCC_ATTRIBUTES = divvy.ComputingConfiguration().keys()
+
+
+dcc_filepath = select_divvy_config(None)
+DCC = divvy.ComputingConfiguration.from_yaml_file(filepath=dcc_filepath)
+DCC_ATTRIBUTES = DCC.keys()
 
 
 @pytest.fixture
 def empty_dcc():
     """Provide the empty/default ComputingConfiguration object"""
-    return divvy.ComputingConfiguration()
+    return divvy.ComputingConfiguration.from_yaml_file(filepath=dcc_filepath)
```

Comment:
this doesn't look like an empty DCC, is this correct?

Reply:
Fixed!